idl/github.com/uber/cadence/replicator.thrift

```diff
@@ -127,15 +127,21 @@ struct ReplicationToken {
   30: optional i64 (js.type = "Long") lastProcessedMessageId
 }
 
+struct SyncShardStatus {
+    10: optional i64 (js.type = "Long") timestamp
+}
+
 struct ReplicationMessages {
   10: optional list<ReplicationTask> replicationTasks
   // This can be different than the last taskId in the above list, because sender can decide to skip tasks (e.g. for completed workflows).
   20: optional i64 (js.type = "Long") lastRetrievedMessageId
   30: optional bool hasMore // Hint for flow control
+  40: optional SyncShardStatus syncShardStatus
```

Comment:
In shard context we have standbyClusterCurrentTime map[string]time.Time
Maybe we should rename it to remoteClusterCurrentTime, as it's not strictly standByCluster anymore.

Reply:
updated the field in shard context